### ISSUE TITLE
Search results are too specific

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/amy#1913]
From @ErinBecker:

> AMY’s search is acting weird. I’m trying to find the curriculum specific onboarding events we did a couple of years ago for genomics, social sciences, and geospatial. All of these have the word “onboarding” in the event name. When I search for “onboarding” I’m taken to [this result](https://amy.carpentries.org/fiscal/organization/datacarpentry-onboarding/) for DC onboarding. Searching for “geospatial” takes me to a list of events in which I’m able to find the [geospatial onboarding event](https://amy.carpentries.org/workshops/event/2018-09-17-geospatial-onboarding/). Trying to find the social sciences onboarding event, I’ve used [socsci](https://amy.carpentries.org/fiscal/membership/227/), [socialsci](https://amy.carpentries.org/requests/training_request/2353/), and [social sciences](https://amy.carpentries.org/fiscal/organization/www.gesis.org/) as search terms, all of which take me to a single result page which is not the one I want. I’m not sure what the problem is mechanically, but something is resulting in AMYs search being too quick to default to what it considers the best match rather than presenting the searcher with a list of options.

**Comment by fmichonneau** [comment]
Related to #1821

**Comment by pbanaszkiewicz** [comment]
I think it's the same bug as #1821. Unfortunately #1821 is going to be released as 3.0, so it's not available yet.

The workaround is to use this URL:
```
https://amy.carpentries.org/dashboard/admin/search/?term=phrase&no_redirect=1
```

The `phrase` should be one word; if you're looking for multiple words, use `+` to connect them, e.g. `https://amy.carpentries.org/dashboard/admin/search/?term=piotr+banaszkiewicz&no_redirect=1`.

Sorry for this inconvenience, @ErinBecker!

**Comment by ErinBecker** [comment]
Thank you @pbanaszkiewicz!
